src/main/java/org/openapitools/codegen/languages/MicronautCodegen.java

```diff
@@ -65,6 +67,11 @@ public class MicronautCodegen extends AbstractJavaCodegen
 
 	public MicronautCodegen() {
 
+		// enable the supported default-codegen features
+
+		supportsAdditionalPropertiesWithComposedSchema = true;
+		useOneOfInterfaces = true;
```

Comment:
Is this overriden from cli options if `cliOptions` is cleared in line #75?

Reply:
Yes, since its not a cliOption, but a feature flag from the DefaultCodegen, that should be set depending on the capabilities of the concrete implementaiton.